settings.gradle.kts

```diff
@@ -8,6 +8,22 @@ pluginManagement {
   }
 }
 
+dependencyResolutionManagement {
```

Comment:
This too would probably go in a plugin (probably replaces repo plugin as a settings plugin)